### DISCUSSION TITLE
imporve model zoo performance

### DISCUSSION THREAD

**Discussion by GitHub-Crow** [facebookresearch/detectron2#3049] [Q&A] [answered]
[faster rcnn](https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md) trained on Pascal VOC in model zoo has  much lower performance than official version, which has  only 51.9mAP, while official version has round 70mAP. is there stronger baseline in model zoo? Or I misread model zoo completely?

**Comment by ppwwyyxx** [comment, marked as answer]
> 70mAP

This probably means AP50, but we use AP[50:95]

**Reply by GitHub-Crow** [reply]
thanks for solving my question. I have considered this, but not sure. 70mAP@[50:95] is exactly too unbelievable...
